gnovm/pkg/gnolang/values.go

```diff
@@ -171,6 +173,9 @@ func (dbv DataByteValue) SetByte(b byte) {
 // or binary operations. When a pointer is to be
 // allocated, *Allocator.AllocatePointer() is called separately,
 // as in OpRef.
+//
+// Since PointerValue is used internally for assignment etc,
+// it MUST stay minimal for computational efficiency.
 type PointerValue struct {
```

Comment:
the original comment on this should be updated as well.
''...A pointer constructed via a &x{} composite lit expression or constructed via new() or make() are independent objects, and have nil Base...."